### PR TITLE
Add export_userScriptEtc()

### PR DESCRIPTION
**Pull request by chiahaoliu**
This PR is to address issue #102. This function bundles everything under `xpdUser/config_base` and `xpdUser/userScript` the export a uncompressed tar file to `xpdUser/Export`.

Code passes `unitttest`.

### CODE REVIEW

**Comment by chiahaoliu** [comment]
updated code has been pushed. Now `dark_base` is removed from directory list and code passes `unittest` as well.

Removing `Export` involves greater degree of refactoring, I will do it in separate PR

File: xpdacq/utils.py [hunk 1]
```diff
@@ -42,8 +42,8 @@ def composition_analysis(compstring):
     fractions = [getfraction(w) for w in namefracs[1::2]]
     return names, fractions
 
-def export_usermetadata():
-    """ functions exporting user meata stored in config_base and userScript under xpdUser directory
+def export_userScript_etc():
```
Comment (sbillinge) [inline xpdacq/utils.py]: Please change the name to ``export_userScriptsEtc()``

too many underscores...

**Comment by chiahaoliu** [comment]
found I haven't fully changed export behavior completely, please don't pull in before updated commit, thanks.

**Comment by chiahaoliu** [comment]
now code are fully updated and passes `unittest` as well.

**Pull request merged**